frontend/src/components/dashboard/geo_location/GroupedTable.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+    <VContainer
+        p-0
+        class="geo-location-table"
+    >
+        <VRow
+            class="header my-3"
+            fluid
+        >
+            <VCol>
+                <slot name="country">
+                    Country
+                </slot>
+            </VCol>
+            <VCol>
+                <slot name="visitors">
+                    Visitors
+                </slot>
+            </VCol>
+            <VCol>
+                <slot name="new-visitors">
+                    New visitors
+                </slot>
+            </VCol>
+            <VCol>
+                <slot name="sessions">
+                    Sessions
+                </slot>
+            </VCol>
+            <VCol>
+                <slot name="bounce-rate">
+                    Bounce rate
+                </slot>
+            </VCol>
+            <VCol>
+                <slot name="avg-session-time">
+                    Avg. session time
+                </slot>
+            </VCol>
+        </VRow>
+        <VDataTable
+            class="caption"
+            hide-default-footer
+            hide-default-header
+            :headers="headers"
+            :items="tableData"
+        />
+    </VContainer>
+</template>
+
+<script>
+    export default {
+        name: 'GroupedTable',
+        props: {
+            items: {
+                type: Array,
+                required: true
+            }
+        },
+        data () {
+            return {
+                headers: [
+                    { text: '', align: 'center', value: 'country' },
+                    { text: '', align: 'center', value: 'visitors' },
+                    { text: '', align: 'center', value: 'new_visitors' },
+                    { text: '', align: 'center', value: 'sessions' },
+                    { text: '', align: 'center', value: 'bounce_rate' },
+                    { text: '', align: 'center', value: 'avg_session_time' },
+                ],
+            };
+        },
+        computed: {
+            tableData () {
```

Comment:
This computed prop is redundant, unless you going to change data format, instead you can just pass items to table component